gufe/tests/test_protocolresult.py

```diff
@@ -0,0 +1,28 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/gufe
+from openff.units import unit
+import pytest
+
+import gufe
+from .test_tokenization import GufeTokenizableTestsMixin
+
+
+class DummyProtocolResult(gufe.ProtocolResult):
+    def get_estimate(self):
+        return self.data['estimate']
+
+    def get_uncertainty(self):
+        return self.data['uncertainty']
+
+
+class TestProtocolResult(GufeTokenizableTestsMixin):
+    cls = DummyProtocolResult
+    key = 'DummyProtocolResult-b7b854b39c1e37feabec58b4000680a0'
+    repr = f'<{key}>'
```

Comment:
ooo, so that my today-I-learned for the day... I should have known it would work, but hadn't thought of it!